operations/last_revision.go

```diff
@@ -21,19 +21,25 @@ func LastRevision() cli.Command {
 	const (
 		regexpVariantsFlagName       = "regex-variants"
 		minSuccessProportionFlagName = "min-success"
+		successfulTasks              = "successful-tasks"
```

Comment:
Can we add something to this name to communicate more strongly what it will do with the successful tasks? 
It's not clear if success is required or if failures are simply not allowed.